test/tests/ucs/test_rackhd_api_ucs.py

```diff
@@ -194,28 +228,24 @@ def test_api_20_ucs_discovery(self):
                                 .format(self.EXPECTED_UCS_NODES, len(self.UCS_NODES)))
         logs.info_1("Found {0} UCS nodes {1}".format(len(self.UCS_COMPUTE_NODES), self.UCS_COMPUTE_NODES))
 
+
     @depends(after=[test_check_ucs_params, test_api_20_ucs_discovery])
     def test_api_20_ucs_catalog(self):
         """
         Tests the UCS Catalog workflow in rackHD
         :return:
         """
-        catalog_workflows = []
         for x in range(len(self.UCS_NODES)):
             postUrl = '/api/2.0/nodes/' + str(self.UCS_NODES[x]["id"]) + "/workflows?name=Graph.Ucs.Catalog"
             header = {"Content-Type": "application/json"}
-            api_data = fit_common.rackhdapi(postUrl, headers=header, action="post")
+            api_data = fit_common.rackhdapi(postUrl, headers=header, action="post", payload={})
             self.assertEqual(api_data['status'], 201,
```

Comment:
Suggestion for later.  This test will assert on the first error found.  For testing, may want to build a list of errors and walk all the USCnodes to see if any worked for status/cataloging.  Similar to what Larry did in this test script -  test/tests/ha/rackdh_ha_resource_test.py